src/lib.rs

```diff
@@ -1116,6 +1116,13 @@ pub fn set_boxed_logger(logger: Box<Log>) -> Result<(), SetLoggerError> {
 /// implementations should provide an initialization method that installs the
 /// logger internally.
 ///
+/// # Availability
+///
+/// This method is available even when the `std` feature is disabled. However,
+/// it is currently unavailable on `thumbv6` targets, which lack support for
+/// some atomic operations which are used by this function. However, even on
```

Comment:
I'd get rid of the However here since the previous sentence started with that as well.